app/domain/mail_relay/lists.rb

```diff
@@ -14,6 +14,8 @@ module MailRelay
   # based on the encoded return path address.
   class Lists < Base
 
+    attr_reader :message, :mail_log
```

Comment:
why is this change required?
this line is already defined in Base and we're inheriting from Base